docs/source/datasets/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _datasets_main:
+
+========
+Datasets
+========
+Here we cover the simulated datasets, which are analogous to 'real world' administrative records such as tax documents and census surveys, that users can generate using :code:`pseudopeople` to test their PRL methods. 
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+
+.. list-table:: **Types of Forms**
+   :header-rows: 1
+   :widths: 20
+
+   * - Name
+   * - | Decennial census
+   * - | American communities survey
+   * - | Current population survey
+   * - | Women, infants, and children survey
+   * - | Social security
+   * - | Tax W2 and 1099 forms
+   * - | Tax 1040 form
+
+Decennial Census
+----------------
+To generate a subset of simulated Census data, you can supply a Census year (i.e., 2020, 2030, 2040), or else use the default configuration. 
```

Comment:
```suggestion
```
I think this sentence is not entirely clear and is perhaps not in the right place, so we should just remove it for now.